v2/event/event_marshal.go

```diff
@@ -120,7 +121,7 @@ func WriteJson(in *Event, writer io.Writer) error {
 		if eventContext.Time != nil {
 			stream.WriteMore()
 			stream.WriteObjectField("time")
-			stream.WriteString(eventContext.Time.String())
+			stream.WriteString(eventContext.Time.Format(time.RFC3339Nano))
```

Comment:
Why are we using Nano here, and the tests don't?

Reply:
If I'm remembering correctly, it was because there were some tests that needed more precision than just "seconds".